### PR TITLE
rework of drps

### PR DESCRIPTION
**Pull request by Haye-ctrl**

### CODE REVIEW

File: assets/default_bundle/drp_60-12Mbits_sin.csv [hunk 1]
```diff
@@ -0,0 +1,3013 @@
+# =======================================
+#               drp_60-12Mbits_sin
+# =======================================
+# Synthetic DataRatePattern
+# This DRP was generated with a falling curve from 60 to 12 Mbits as base,
+# noise is added, sin waves of different frequencies are added.
```
Comment (merren-fx) [inline assets/default_bundle/drp_60-12Mbits_sin.csv]: Typo correction
```suggestion
# noise is added, sine waves of different frequencies are added.
```

File: assets/default_bundle/drp_bratislava.csv [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # =======================================
-#       drp_bratislava / drp_prague       
+#       drp_bratislava / drp_prague
 # =======================================
 # eNodeB-logs
 # This DRP was extracted from the logs of an eNodeB.
```
Comment (merren-fx) [inline assets/default_bundle/drp_bratislava.csv:5]: Please add a line describing that it was modified from a 'real' pattern

File: assets/default_bundle/drp_munich_autobahn.csv [hunk 3]
```diff
@@ -1,4294 +1,2988 @@
 # =======================================
-#           drp_munich_autobahn                 
+#           drp_munich_autobahn
 # =======================================
 # iperf3 DataRatePattern (UDP)
 # This DRP was recorded both on a Autobahn and a Bundesstraße
 # in the wieder area of munich.
-#
```
Comment (merren-fx) [inline assets/default_bundle/drp_munich_autobahn.csv]: Please add a line describing that it was modified from a 'real' pattern

File: assets/default_bundle/drp_munich_outskirt.csv [hunk 4]
```diff
@@ -1,6171 +1,6116 @@
 # =======================================
-#           drp_munich_autobahn                 
+#           drp_munich_autobahn
 # =======================================
 # iperf3 DataRatePattern (UDP)
 # This DRP was recorded on a smaller Umgehungsstraße
 # in the wieder area of munich.
-#
```
Comment (merren-fx) [inline assets/default_bundle/drp_munich_outskirt.csv]: Please add a line describing that it was modified from a 'real' pattern

File: assets/default_bundle/drp_munich_village.csv [hunk 5]
```diff
@@ -1,1629 +1,1225 @@
 # =======================================
-#           drp_munich_autobahn                 
+#           drp_munich_autobahn
 # =======================================
 # iperf3 DataRatePattern (UDP)
 # This DRP was recorded on a city/village street
 # in the wieder area of munich.
-#
```
Comment (merren-fx) [inline assets/default_bundle/drp_munich_village.csv]: Please add a line describing that it was modified from a 'real' pattern

**Review by merren-fx — CHANGES_REQUESTED** [review]
Looks good overall, only descriptions of patterns need modification.

File: assets/default_bundle/drp_60-12Mbits_sin.csv (hunk 1 above)
Comment (Haye-ctrl) [reply assets/default_bundle/drp_60-12Mbits_sin.csv]: done

File: assets/default_bundle/drp_bratislava.csv (hunk 2 above)
Comment (Haye-ctrl) [reply assets/default_bundle/drp_bratislava.csv:5]: done

File: assets/default_bundle/drp_munich_autobahn.csv (hunk 3 above)
Comment (Haye-ctrl) [reply assets/default_bundle/drp_munich_autobahn.csv]: done

File: assets/default_bundle/drp_munich_outskirt.csv (hunk 4 above)
Comment (Haye-ctrl) [reply assets/default_bundle/drp_munich_outskirt.csv]: done

File: assets/default_bundle/drp_munich_village.csv (hunk 5 above)
Comment (Haye-ctrl) [reply assets/default_bundle/drp_munich_village.csv]: done

Comment (Haye-ctrl) [reply assets/default_bundle/drp_munich_village.csv]: Some more typos in the comments have been corrected.

**Review by merren-fx — APPROVED** [review]

**Pull request merged**